torchao/utils.py

```diff
@@ -694,3 +696,11 @@ def check_xpu_version(device, version="2.8.0"):
 TORCH_VERSION_AFTER_2_4 = _torch_version_at_least("2.4.0.dev")
 TORCH_VERSION_AFTER_2_3 = _torch_version_at_least("2.3.0.dev")
 TORCH_VERSION_AFTER_2_2 = _torch_version_at_least("2.2.0.dev")
+
+
+def is_package_at_least(package_name: str, min_version: str):
+    package_exists = importlib.util.find_spec(package_name) is not None
+    if not package_exists:
+        return False
+
+    return version(package_name) >= min_version
```

Comment:
do version conversion on both

Reply:
`version` is called on the package, min_version is just a string

you probably refers to `version.parse` like https://github.com/huggingface/transformers/blob/3b3ebcec4077f124f2cd0ec3cd5d028dc352a3e5/src/transformers/utils/quantization_config.py#L566